apis/digitalocean_api.php

```diff
@@ -50,12 +50,12 @@ private function makeGetConnection($action) {
         $ch = curl_init();
         curl_setopt($ch, CURLOPT_HTTPHEADER, array(
             'Content-Type: application/json',
-            'Accept: application/json'
+            'Accept: application/json',
+			'Authorization: Bearer ' . $this->apiKey
```

Comment:
Can you fix the tabbing here? Each array element on its own line should be vertically aligned by the same indentation.